nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/vision/AbstractStartGcpVisionOperation.java

```diff
@@ -70,7 +92,15 @@ public void onStopped() throws IOException {
     }
 
     private InputStream getInputStreamFromProperty(ProcessContext context, FlowFile flowFile) {
-        return new ByteArrayInputStream(context.getProperty(getJsonPayloadPropertyDescriptor()).evaluateAttributeExpressions(flowFile).getValue().getBytes(StandardCharsets.UTF_8));
+        Map<String, String> attributes = new HashMap<>();
+        attributes.put(OUTPUT_BUCKET.getName(), getAttributeValue(context, flowFile, OUTPUT_BUCKET.getName()));
+        attributes.put(FEATURE_TYPE.getName(), getAttributeValue(context, flowFile, FEATURE_TYPE.getName()));
+        return new ByteArrayInputStream(context.getProperty(getJsonPayloadPropertyDescriptor())
+                .evaluateAttributeExpressions(flowFile, attributes).getValue().getBytes(StandardCharsets.UTF_8));
```

Comment:
Splitting this declaration into multiple lines would make it easier to follow.
```suggestion
        final PropertyValue jsonPropertyValue = context.getProperty(getJsonPayloadPropertyDescriptor());
        final String jsonPayload = jsonPropertyValue.evaluateAttributeExpressions(flowFile, attributes).getValue();
        return new ByteArrayInputStream(jsonPayload.getBytes(StandardCharsets.UTF_8));
```